ReactAndroid/build.gradle

```diff
@@ -192,6 +206,20 @@ task downloadNdkBuildDependencies {
     dependsOn(downloadGlog)
 }
 
+def findNodeModulePath(baseDir, packageName) {
```

Comment:
Have you considered invoking the actual node binary to resolve the path instead? This seems good enough but I’m pretty sure there are a lot of edge cases in the node resolution algorithm

Reply:
Many of the edge cases in Node's module resolution algorithm are around finding the main module (needing to look at package.json["main"] and defaulting to index.js, for example) and handling `@scoped/packages`, neither of which we care about here. The algorithm for finding a package is actually pretty simple if you're not trying to super-optimize it - https://github.com/nodejs/node/blob/6db45bf7def22eedfd95dac95713e850c366b169/lib/internal/modules/cjs/loader.js#L584-L618